src/main/kotlin/gg/skytils/skytilsmod/gui/profile/components/ItemComponent.kt

```diff
@@ -38,7 +38,7 @@ class ItemComponent(val state: State<ItemStack>) : UIComponent() {
         super.draw(matrixStack)
         matrixStack.push()
         matrixStack.translate(getLeft(), getTop(), 100f)
-        matrixStack.scale(getWidth() / 16f, getHeight() / 16f, 0f)
```

Comment:
Is this change necessary? It should adjust the skull to be the appropriate size for an arbitrarily sized `ItemComponent`

Reply:
This scaling breaking skull textures. I didnt see any problem in scaling when i tried it without this. (as seen in the images above) Thats why i didnt think there was a need to alternative scaling. Another e.g. using ItemComponent is
**SuperboomToast (Old -> New)**
![toast](https://github.com/Skytils/SkytilsMod/assets/10743391/b0608d48-9c36-439a-98fc-96ca840f8f88)

Reply:
Those all used 16 pixel scalings, could you see if it still works with something like 20 pixels or larger like 32?

Reply:
`matrixStack.scale(getWidth() / 32f, getHeight() / 32f, 0f)`
![image](https://github.com/Skytils/SkytilsMod/assets/10743391/1e7b414d-b58a-4d40-9afd-255991767dd9)
`matrixStack.scale(getWidth() / 20f, getHeight() / 20f, 0f)`
![image](https://github.com/Skytils/SkytilsMod/assets/10743391/61ab8c20-3a94-4a6c-9ad5-e63ffa8b32ac)
`matrixStack.scale(getWidth() / 4f, getHeight() / 4f, 0f)`
![image](https://github.com/Skytils/SkytilsMod/assets/10743391/5a24081d-a29e-4a98-bacf-2a27e12d0966)
😂

Reply:
Hm very interesting. I'll have to investigate this further later.